boilerplate/app/components/Toggle.tsx

```diff
@@ -261,26 +262,26 @@ function Checkbox(props: ToggleInputProps) {
     disabled && colors.palette.neutral400,
     status === "error" && colors.errorBackground,
     colors.palette.neutral200,
-  ].filter(Boolean)[0]
+  ].filter((item): item is string => Boolean(item))[0]
```

Comment:
you explained this to me earlier, i wonder if there is a way to still do `.filter(Boolean)`? 🧐

Reply:
Hmm not sure because you have to help along typing the item

Reply:
found this solution online: https://www.karltarvas.com/typescript-array-filter-boolean.html.

this commit follows the example: https://github.com/infinitered/ignite/pull/2446/commits/c24038569a7decc33f5107a833a4732a446b7c01